QualityControl/public/common/filters/filterViews.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { filters } from '../../../common/filters/filter.js';
```

Comment:
I think this import is incorrect. Is it not actually `../../common/filters/filter.js`? Which if you are looking at it is actually just `./filter.js`?